ionoscloud/resource_server_test.go

```diff
@@ -805,6 +805,49 @@ func TestAccServerWithLabels(t *testing.T) {
 	})
 }
 
+func TestAccServerBootDeviceSelection(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		ExternalProviders: randomProviderVersion343(),
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckServerDestroyCheck,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMultipleBootDevices,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrPair(constant.ServerResource+"."+constant.ServerTestResource, "boot_volume", constant.ServerResource+"."+constant.ServerTestResource, "inline_volume_ids.0"),
+					resource.TestCheckResourceAttr(constant.ServerResource+"."+constant.ServerTestResource, "boot_cdrom", ""),
+				)},
+			// The server object is updated 'outside' of the resource, so the state of the server resource won't be won't be refreshed in the same step
```

Comment:
```suggestion
			// The server object is updated 'outside' of the resource, so the state of the server resource won't be  refreshed in the same step
```

Reply:
typo fixed in - [6dd8e09](https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/507/commits/6dd8e09c4de1c678f0c2f24d6ee64e48aa1b5f4a)